src/main/resources/i18n/Messages_en.properties

```diff
@@ -167,6 +167,14 @@ notification.observation.upcoming.email.body.text.3=To complete observations, do
 notification.observation.upcoming.email.subject=Upcoming Observation
 notification.organization.email.buttonIntro=Please click the button below to go to your Terraware account where you can view the organization.
 notification.organization.email.linkIntro=Please click the link below to go to your Terraware account where you can view the organization.
+notification.participantProjectSpecies.added.ctaLabel=Review in Terraware
+notification.participantProjectSpecies.added.longBody={0} has submitted {1} for use for {2} and is ready for approval.
+notification.participantProjectSpecies.added.shortBody={0} has been submitted for use for {1}.
+notification.participantProjectSpecies.added.subject=A species has been added to a project for {0}.
+notification.participantProjectSpecies.edited.ctaLabel=Review in Terraware
+notification.participantProjectSpecies.edited.longBody={0} has edited {1} and is ready for approval.
+notification.participantProjectSpecies.edited.shortBody={0} has been edited and is ready for approval.
+notification.participantProjectSpecies.edited.subject=An approved species has been edited for {0}.
```

Comment:
This string naming convention is new, yes? Would it be worth having a separate discussion on changing the naming of notification strings? I think it's preferable to have a consistent naming convention.

I guess my preference (but happy to discuss it) would be to use the existing naming convention for these strings and if we decide to switch to the scheme in this PR, we can update all the strings at once.

Reply:
Yea I wasn't really sure what the correct way to do it was with notifications that are both emailed and sent in app, without duplicating entries, so I just made it up. I will ask in Slack and figure out the correct way.

Reply:
Going to put this back to the pre-existing conventions